pkgs/development/libraries/x265/default.nix

```diff
@@ -67,7 +54,9 @@ stdenv.mkDerivation rec {
     rm $out/lib/*.a
   '';
 
-  nativeBuildInputs = [ cmake yasm ];
+  nativeBuildInputs = [ cmake nasm
+  ] ++ stdenv.lib.optional numaSupport numactl;
```

Comment:
(nitpick)
Slightly unusual style. I guess you can keep all of it on a single line